src/test/kotlin/net/atos/zac/app/admin/ReferenceTableRestServiceTest.kt

```diff
@@ -110,4 +114,97 @@ class ReferenceTableRestServiceTest : BehaviorSpec({
             }
         }
     }
+
+    Given("An existing reference table with one value") {
+        val referenceTable = createReferenceTable()
+        val updatedReferenceTable = createReferenceTable()
+        val updatedReferenceTableSlot = slot<ReferenceTable>()
+        every { policyService.readOverigeRechten().beheren } returns true
+        every { referenceTableService.readReferenceTable(referenceTable.id!!) } returns referenceTable
+        every {
+            referenceTableAdminService.updateReferenceTable(capture(updatedReferenceTableSlot))
+        } returns updatedReferenceTable
+
+        When("the reference table is updated with two new values and a new name") {
+            val restReferenceTableUpdate = createRestReferenceTableUpdate(
+                naam = "dummyUpdatedName",
+                waarden = listOf(
+                    createRestReferenceTableValue(
+                        name = "dummyWaarde100"
+                    ),
+                    createRestReferenceTableValue(
+                        name = "dummyWaarde101"
+                    )
+                )
+            )
+
+            val updatedRestReferenceTable = referenceTableRestService.updateReferenceTable(
+                id = referenceTable.id!!,
+                restReferenceTableUpdate
+            )
+
+            Then("the reference table is updated successfully and only contains the two new values") {
```

Comment:
huh? 
Given "An existing reference table with one value"
When "the reference table is updated with two new values..."
Then "the reference table is updated successfully and only contains the two new values"

I don't get where the original value has gone... I'd expect 3 values from the description

Reply:
I think I understand from reading a bit more, but it's not clear from the "then" statement...

Reply:
>I don't get where the original value has gone... I'd expect 3 values from the description

No, that is not how the update endpoint works. The update endpoint overrides the reference table values (and name) completely. That's how it now works and I think that's ok. Also, not worth changing that behavior I think. It's not like a merge or anything.